winsup/cygwin/lib/cygwin_crt0.c

```diff
@@ -16,6 +16,12 @@ extern void _dll_crt0 ()
 void
 cygwin_crt0 (MainFunc f)
 {
+  static int init = 0;
+  if (init)
```

Comment:
This is also quite dangerous and not needed with `use_internal = true;`

Reply:
The same here.